docs/script.rst

```diff
@@ -414,77 +409,77 @@ Remove first found numbers followed by a whitespace character::
+files in the result list, add the optional parameter ``hidden``::
 
   listWithHidden = file('some/path/*.fa', hidden: true)
 
-The list of available options is shown below:
+Here are ``file``'s available options:
 
 =============== ===================
 Name            Description
 =============== ===================
-glob            When ``true`` interprets characters ``*``, ``?``, ``[]`` and ``{}`` as glob wildcards, otherwise handles them as normal characters (default: ``true``)
+glob            When ``true`` interpret characters ``*``, ``?``, ``[]`` and ``{}`` as glob wildcards, otherwise handle them as normal characters (default: ``true``)
 type            Type of paths returned, either ``file``, ``dir`` or ``any`` (default: ``file``)
-hidden          When ``true`` includes hidden files in the resulting paths (default: ``false``)
+hidden          When ``true`` include hidden files in the resulting paths (default: ``false``)
 maxDepth        Maximum number of directory levels to visit (default: `no limit`)
-followLinks     When ``true`` it follows symbolic links during directories tree traversal, otherwise they are managed as files (default: ``true``)
+followLinks     When ``true`` follow symbolic links during directory tree traversal, otherwise treat them as files (default: ``true``)
 =============== ===================
 
 
 .. tip:: If you are a Java geek you will be interested to know that the ``file`` method returns a
   `Path <http://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html>`_ object, which allows
-  you to use the usual methods as you would in a Java program.
+  you to use the usual methods you would in a Java program.
 
 See also: :ref:`Channel.fromPath <channel-path>` .
 
 Basic read/write
 ------------------
 
-Given a file variable, declared by using the ``file`` method as shown in the previous example, reading a file
+Given a file variable, declared using the ``file`` method as shown in the previous example, reading a file
 is as easy as getting the value of the file's ``text`` property, which returns the file content
-as a string value. For example::
+as a string value::
 
   print myFile.text
 
 
-In the same way you can save a string value to a file by simply assigning it to the file's ``text`` property,
-as shown below::
+Similarly, you can save a string value to a file by simply assigning it to the file's ``text`` property::
 
   myFile.text = 'Hello world!'
 
 
-.. note:: The actual file content is overwritten by the assignment operation. If the file does not exist, it is created
-  implicitly by the assignment operation.
+.. note:: Existing file content is overwritten by the assignment operation, which also implicitly creates
+          files that do not exist.
 
-In order to append a string value to a file without erasing the actual content, you can use the ``append`` method::
+In order to append a string value to a file without erasing existing content, you can use the ``append`` method::
```

Comment:
I know what you mean by "actual", but it's strange phrasing in English. I think "exiting", or "current" (probably closer to what you think of when translating "actual") is better.